apps/api/src/app/annotations/entities/annotation.entity.ts

```diff
@@ -21,21 +21,19 @@ export class Annotation extends BaseEntity {
   @Field(() => Int, { nullable: true, description: 'Annotation End Seconds' })
   end?: number;
 
-  @JoinColumn({ name: 'markerId' })
```

Comment:
Why is this not necessary any more?

Reply:
You can omit JoinColumn for ManyToOne / OneToMany, see here https://typeorm.io/many-to-one-one-to-many-relations